plugins/cratedb/database_credentials.go

```diff
@@ -0,0 +1,82 @@
+package cratedb
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DatabaseCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.DatabaseCredentials,
+		DocsURL:       sdk.URL("https://crate.io/docs/crate/crash/en/latest/run.html#environment-variables"),
+		ManagementURL: sdk.URL("https://console.cratedb.cloud/account/settings"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Host,
+				MarkdownDescription: "CrateDB host to connect to.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.Port,
+				MarkdownDescription: "Port used to connect to CrateDB.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.User,
+				MarkdownDescription: "CrateDB user to authenticate as.",
+				Optional:            true,
```

Comment:
Very well done on adding all these optional setting fields, it brings a lot of value to future users of this plugin!

Reply:
😀😀